### ISSUE TITLE
Objects completely not rendering + start time after end time 

### ISSUE THREAD

**Issue by AwesomeGamer89** [thebetioplane/osuReplayEditorV3#39]
https://user-images.githubusercontent.com/22873716/151690682-79551927-f853-465d-ae60-709022a510b7.mp4

Beatmap: https://osu.ppy.sh/beatmapsets/257607#osu/878369

**Comment by thebetioplane** [comment]
I've found the problem, the beatmap uses decimal numbers in it's timing points
```
[TimingPoints]
997.60640255051,270.00027000027,4,1,0,100,1,0
107917,-90.9090909090909,4,1,0,100,0,0
146797,-100,4,1,0,100,0,0
163807,-86.9565217391304,4,1,0,100,0,0
200797,-76.9230769230769,4,1,0,100,0,0
243997,-66.6666666666667,4,1,0,100,0,0
```
I referenced the osu!wiki - [.osu file format, section timing points](https://osu.ppy.sh/wiki/en/Client/File_formats/Osu_%28file_format%29#timing-points) - which says these should be integers. Either this beatmap is older than these standards or the beatmapper edited these by hand.

Weird, but if osu! can handle it then I will fix it to allow decimals.
